packages/typescript/test/fixtures/incremental-watch-off/dist/.tsbuildinfo

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
Should this file be
1. checked in?
2. published as a build artifact? (it is not excluded from "files" in `package.json`)

Reply:
I'm not the one to make the decision so I'll only add some information that lead me to include it.  

- `packages/typescript/test/fixtures/incremental/dist/.tsbuildinfo` is checked in. 

- The repo's `.gitignore` has explicitly set `!packages/typescript/test/fixtures/**/dist`.

Reply:
Ah I am sorry, I just saw `dist` but overlooked that it was in the `test` folder. In that case everything should be fine.